discovery/vultr/vultr.go

```diff
@@ -0,0 +1,212 @@
+// Copyright 2022 The Prometheus Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package vultr
+
+import (
+	"context"
+	"fmt"
+	"net"
+	"net/http"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/go-kit/log"
+	"github.com/prometheus/common/config"
+	"github.com/prometheus/common/model"
+	"github.com/prometheus/common/version"
+	"github.com/vultr/govultr/v2"
+
+	"github.com/prometheus/prometheus/discovery"
+	"github.com/prometheus/prometheus/discovery/refresh"
+	"github.com/prometheus/prometheus/discovery/targetgroup"
+)
+
+const (
+	vultrInstanceLabel             = model.MetaLabelPrefix + "vultr_instance_"
+	vultrInstanceLabelID           = vultrInstanceLabel + "id"
+	vultrInstanceLabelLabel        = vultrInstanceLabel + "label"
+	vultrInstanceLabelOS           = vultrInstanceLabel + "os"
+	vultrInstanceLabelOSID         = vultrInstanceLabel + "os_id"
+	vultrInstanceLabelRegion       = vultrInstanceLabel + "region"
+	vultrInstanceLabelPlan         = vultrInstanceLabel + "plan"
+	vultrInstanceLabelMainIP       = vultrInstanceLabel + "main_ip"
+	vultrInstanceLabelMainIPV6     = vultrInstanceLabel + "main_ipv6"
+	vultrInstanceLabelPrivateIPv4  = vultrInstanceLabel + "internal_ip"
```

Comment:
```suggestion
	vultrInstanceLabelInternalIP  = vultrInstanceLabel + "internal_ip"
```

Reply:
Completely forgot about docs! Added those in and also updated the label for bandwidth to match the API response